### PR TITLE
103 append toi connect time delay algorithm dgr

### PR DESCRIPTION
**Pull request by d-gregorczyk**

### CODE REVIEW

**Comment by d-gregorczyk** [comment]
Please ignore everything below the asciidoc-converter folder unless you are familiar with Kotlin ;-)

**Review by riechkathrin — APPROVED** [review]

File: SDPi_Supplement/asciidoc-converter/src/main/kotlin/org/sdpi/asciidoc/Util.kt [hunk 1]
```diff
@@ -1,21 +1,20 @@
 package org.sdpi.asciidoc
 
 import org.apache.logging.log4j.kotlin.loggerOf
-import org.asciidoctor.ast.Section
 import org.asciidoctor.ast.StructuralNode
+import org.sdpi.asciidoc.model.RequirementLevel
```
Comment (ToddCooper) [inline SDPi_Supplement/asciidoc-converter/src/main/kotlin/org/sdpi/asciidoc/Util.kt:5]: What was the need to include the .RequirementLevel support in the asciidoc-converter tool?

**Review by ToddCooper — APPROVED** [review]
This should get merged today if possible.  THEN Create a new issue & branch to finalize the sdpi_requirement model, documentation and implementations.

File: SDPi_Supplement/asciidoc-converter/src/main/kotlin/org/sdpi/asciidoc/BlockAttribute.kt [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 /**
- * Known ASCIIDoc attributes.
+ * Known ASCIIDoc and SDPi attributes.
  */
 enum class BlockAttribute(val key: String) {
     ID("id"),
-    TITLE("title");
+    TITLE("title"),
+    ROLE("role"),
+    REQUIREMENT_LEVEL("sdpi_req_level"),
```
[the remark this answers is not in the document]
Comment (d-gregorczyk) [reply SDPi_Supplement/asciidoc-converter/src/main/kotlin/org/sdpi/asciidoc/BlockAttribute.kt:10]: Nope, this is not supposed to change once we're done defining the necessary building blocks to describe requirements. Until then, we might have to adapt the converter to the RI model.

File: SDPi_Supplement/asciidoc-converter/src/main/kotlin/org/sdpi/asciidoc/Util.kt (hunk 1 above)
Comment (d-gregorczyk) [reply SDPi_Supplement/asciidoc-converter/src/main/kotlin/org/sdpi/asciidoc/Util.kt:5]: This is supposed to be meta information, which can also be used to verify the existence of requirement keywords in the actual requirement texts.

**Pull request merged**